scripts/js/utils.js

```diff
@@ -468,7 +468,7 @@ function doLogout() {
     url: "/api/auth",
     method: "DELETE",
   }).always(function () {
-    location.reload();
+    globalThis.location = "/admin/login";
```

Comment:
For _most_ this location is fine but keep in mind that the `/admin/` part of the path is configurable via `webserver.paths.webhome`

![image](https://github.com/user-attachments/assets/44265086-0607-4f13-9610-f018511e056b)

Reply:
Yup, hence why I'm not sure if this is the best approach. I still think we should use the LUA template for this action more. Similarly to what we did with PHP.

Reply:
We could probably hack something around and pass the URL to the logout button's `data` attribute or something, but I'm still not sure if it's the best approach...

Reply:
> We could probably hack something around and pass the URL

If this is really needed, we can use `pihole.webhome()` in Lua code to set an attribute in any HTML tag. 

Maybe set `data-webhome="<?= pihole.webhome() ?>"` in `<body>` tag (or any other attribute name).